openff/qcsubmit/validators.py

```diff
@@ -259,3 +259,27 @@ def check_valence_connectivity(molecule: qcel.models.Molecule) -> qcel.models.Mo
         )
 
     return molecule
+
+
+def check_allowed_elements(element: Union[str, int]) -> Union[str, int]:
+    """
+    Check that each item can be cast to a valid element.
+
+    Parameters:
+        element: The element that should be checked.
+
+    Raises:
+        ValueError: If the element number or symbol passed could not be converted into a valid element.
+    """
+    from simtk.openmm.app import Element
+
+    if isinstance(element, int):
+        return element
+    else:
+        try:
+            _ = Element.getBySymbol(element)
+            return element
+        except KeyError:
+            raise ValueError(
+                f"An element could not be determined from symbol {element}, please enter symbols only."
+            )
```

Comment:
```suggestion
    try:
        _ = Element.getBySymbol(element)
        return element
    except KeyError:
        raise ValueError(
            f"An element could not be determined from symbol {element}, please enter symbols only."
        )
```

We can get away without the `else` statement here as the `if` statement returns when it is triggered. This lets us un-nest the larger code block which usually helps with readability 🙂

Reply:
Ah, good spot thanks!